src/gasPriceOracle/adapters/arbitrum.ts

```diff
@@ -19,7 +23,7 @@ export async function eip1559(provider: providers.Provider, chainId: number): Pr
 
   // eip1559() sets maxFeePerGas = lastBaseFeePerGas + maxPriorityFeePerGas, so revert that.
   // The caller may apply scaling as they wish afterwards.
-  const maxFeePerGas = _maxFeePerGas.sub(maxPriorityFeePerGas).add(bnOne);
+  const maxFeePerGas = _maxFeePerGas.mul(markup).sub(maxPriorityFeePerGas).add(bnOne);
```

Comment:
Does this work for `0 < markup < 1` ? When I test it locally it causes an underflow locally because ethers BigNumber doesn't like numbers < 1.0.

```
> y = ethers.BigNumber.from("1000000000")
BigNumber { _hex: '0x3b9aca00', _isBigNumber: true }
> y.mul(0.9)
Uncaught:
Error: underflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-underflow ] (fault="underflow", operation="BigNumber.from", value=0.9, code=NUMERIC_FAULT, version=bignumber/5.7.0)
```

Reply:
good point, floored it at 1.0 instead